talk/C++Course.tex

```diff
@@ -2,12 +2,11 @@
 \usetheme{Warsaw}
 \useoutertheme{split}
 
-%\includeonly{python/python}
-
 \input{setup}
 
 \newboolean{onlybasics}
 \setboolean{onlybasics}{false}
+\input{onlybasics}
```

Comment:
Could you please test first, if the file exists? Something like:
```suggestion
\IfFileExists{onlybasics.tex}{\input{onlybasics}}{}
```
This way, I do not need to create the file myself if I use an IDE like `TexStudio` or an interface such as Overleaf.